tests/unit/test_lib_per_app_requires.py

```diff
@@ -77,27 +77,28 @@ def test_ingress_app_requirer_related(requirer: IngressPerAppRequirer, harness,
 @pytest.mark.parametrize(
     "auto_data, ok",
     (
-        ((True, 42), False),
-        ((10, False), False),
-        ((10, None), False),
-        (("foo", 12), True),
+        ((True, "example.com", 42), False),
+        ((10, "example.com", False), False),
+        ((10, "example.com", None), False),
```

Comment:
we should probably add a case in which everything is OK except for the IP, such as

`("foo", "not.an.ip", 12)`

to make sure we 're not only testing the other failures

Reply:
I added that case, thanks!